python/pyspark/ml/clustering.py

```diff
@@ -347,14 +418,20 @@ class KMeans(JavaEstimator, HasDistanceMeasure, HasFeaturesCol, HasPredictionCol
     >>> df = spark.createDataFrame(data, ["features"])
     >>> kmeans = KMeans(k=2, seed=1)
     >>> model = kmeans.fit(df)
+    >>> model.getDistanceMeasure()
+    'euclidean'
+    >>> model.setPredictionCol("newPrediction")
+    KMeans...
```

Comment:
KMeansModel...

Reply:
It is ```KMeans_3487dfaa7c0e```

Reply:
This maybe a little bug, `KMeans` & `KMeansModel` should have different uid (like `LogisticRegression` & `LogisticRegressionModel`). This issue seems also happen in other place. But we can leave it here.